koap-core/src/commonMain/kotlin/Message.kt

```diff
@@ -109,6 +110,72 @@ sealed class Message {
             ) : Format()
         }
 
+        data class UnassignedOption(
```

Comment:
These are members of the `Option` sealed class, I feel the `Option` postfix is redundant. If consumers want the verbosity, they can use them as, for example, `Option.Unassigned`.

```suggestion
        data class Unassigned(
```

Reply:
yes, of course. I thought those names were a bit too generic, but they are already inside Option!

Reply:
updated